src/app/workflow/paramfiles/paramfiles.component.ts

```diff
@@ -14,7 +14,6 @@
  *    limitations under the License.
  */
 import { AfterViewChecked, Component, ElementRef, Input } from '@angular/core';
-import { Component, Input, OnInit } from '@angular/core';
```

Comment:
Should be the above line that's to be removed, same applies for many other files

Reply:
Not exactly, the OnInit for example isn't used.

Reply:
then remove OnInit also